### PR TITLE
Adding option for cache reading and specify openshift version.

### PR DESCRIPTION
**Pull request by kwoodson**

### CODE REVIEW

File: bin/ohi [hunk 1]
```diff
@@ -85,23 +92,27 @@ class Ohi(object):
...
-        parser.add_argument('-l', '--user', action='store', default=None,
-                               help='username')
+        parser.add_argument('-l', '--user', action='store', default=None, help='username')
 
+        parser.add_argument('-c', '--cache-only', action='store_true', default=False, 
+                            help='Retrieve the host inventory by cache only. Default is false.')
 
-        self.args = parser.parse_args()
+        parser.add_argument('-o', '--openshift-version', action='store', required=True,
+                            help='Specify the openshift account version.')
```
Comment (twiest) [inline bin/ohi]: "account version" ?

I see what you are going for, but I don't think account version is correct.

I think just openshift version is probably good.

**Comment by kwoodson** [comment]
ok to test

**Comment by twiest** [comment]
:+1:

**Comment by twiest** [comment]
:+1:

**Pull request merged**